### ISSUE TITLE
Have you sovled? i meet this issue @BootsofLagrangian thanks for taking a look!

### ISSUE THREAD

**Issue by wuliebucha** [kohya-ss/sd-scripts#976]
@BootsofLagrangian thanks for taking a look! 

Settings:
```
network_dim=16
network_alpha=8
unet_lr=0.5
text_encoder_lr=0.5
optimizer_type="DAdaptation"
optimizer_args=["decouple=True", "weight_decay=1.0"]
lr_scheduler="constant"
min_snr_gamma=5
```

tried weight decay 0.5, 1.5 as well.

![Screenshot 2023-04-21 at 15-23-14 TensorBoard](https://user-images.githubusercontent.com/15027/233719780-c3f4f1f7-0943-41d5-a89c-22e8285fba83.png)
![Screenshot 2023-04-21 at 15-33-43 TensorBoard](https://user-images.githubusercontent.com/15027/233719935-23a6118d-a44c-4d5a-b885-b38e3a2658a7.png)

And the last light blue is without min_snr_gamma but has the same problem

![Screenshot 2023-04-21 at 15-36-17 TensorBoard](https://user-images.githubusercontent.com/15027/233720401-8578437d-ec52-4fd1-9322-da39f9adb8e7.png)

once it starts going up it starts producing noise and goes up fast and never recovers. Using the same dataset and settings (changing the learning rate, and low or no weight decay) with AdamW produces good results so its within the reported settings being changed. 

Example of the noise: 
![landscape-5-2023-04-21-152608-ce5def3a_20230421153540_000050_03_2661845567](https://user-images.githubusercontent.com/15027/233720477-a114e9d1-257f-4f04-b661-556983e39c31.png)
![landscape-5-2023-04-21-151628-dfe93462_20230421152026_000020_07_2661845567](https://user-images.githubusercontent.com/15027/233720513-17164e5e-956c-4e9d-b422-62067b48aa7a.png)

Edit: Also noting I'm running `batch size: 2, gradient accumulation steps: 24` in these tests. Maybe this is impacting how it is working?

_Originally posted by @rockerBOO in https://github.com/kohya-ss/sd-scripts/issues/181#issuecomment-1518259458_
